include/models/integration_account.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef _integration_account_hpp_
+#define _integration_account_hpp_
+
+#include <models/entity.hpp>
+
+namespace disccord
+{
+    namespace models
+    {
+        class integration_account : public model
+        {
+            public:
+                integration_account();
+                virtual ~integration_account();
+
+                virtual void decode(web::json::value json) override;
+
+                std::string get_id();

```

Comment:
ah yes oops, yesterday i was tryna find all the ones that shouldve been ulong, guess I missed this one. but yeah, it should inherit entity